db/app.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const app = express();
+app.use(express.json());
```

Comment:
Is this line of code doing anything at the moment? If not, please remove until it's useful 🙂

Reply:
Thanks for the feedback c: I thought the first 2 lines were needed at least for the routes to work?

Reply:
I have understood now. Thanks!